### PR TITLE
Feature routing

### PR DESCRIPTION
**Pull request by martijnjanssen**
The webpage now opens on the correct corresponsing routes when opening the url. Starting at the "home" view, without a url. The other pages have their url set to the page-contents.

### CODE REVIEW

File: src/symposium-app/symposium-app.html [hunk 1]
```diff
@@ -110,11 +117,11 @@
         </app-toolbar>
 
         <!-- Nav on mobile: side nav menu -->
-        <paper-menu selected="{{selected}}" on-tap="scrollTop">
-            <paper-item>Home </paper-item>
-            <paper-item>Program</paper-item>
-            <paper-item>Location</paper-item>
-            <paper-item>Committee</paper-item>
+        <paper-menu selected="{{data.page}}" attr-for-selected="key" on-tap="scrollTop">
+            <paper-item key="">Home</paper-item>
```
Comment (TimvdLippe) [inline src/symposium-app/symposium-app.html]: Since `key` is not a property of `paper-item`, it is better if you use `data-key` per the HTML spec

**Review by TimvdLippe — COMMENTED** [review]
Have not tested locally, but should be working just fine I suppose. Just 1 small comment

File: src/symposium-app/symposium-app.html (hunk 1 above)
Comment (martijnjanssen) [reply src/symposium-app/symposium-app.html]: So for items that are not a property of the element itself, it is better to add a `data-` prefix to the property? Is that correct?

Comment (TimvdLippe) [reply src/symposium-app/symposium-app.html]: Yes. See https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#embedding-custom-non-visible-data-with-the-data-attributes

**Review by TimvdLippe — APPROVED** [review]
Statically looks good, Martijn tested it. Merging!

**Pull request merged**